src/community/advanced-data-card.css.ts

```diff
@@ -0,0 +1,196 @@
+import {style, styleVariants} from '@vanilla-extract/css';
+import {sprinkles} from '../sprinkles.css';
+import * as mq from '../media-queries.css';
+import {vars} from '../skins/skin-contract.css';
+import {applyAlpha} from '../utils/color';
+
+export const paddingX = style([
+    sprinkles({
+        paddingX: 16,
+    }),
+    {
+        '@media': {
+            [mq.desktopOrBigger]: {
+                paddingLeft: 24,
+                paddingRight: 24,
+            },
+        },
+    },
+]);
+
+export const actions = style([
+    paddingX,
+    sprinkles({
+        display: 'flex',
+        flex: 1,
+        justifyContent: 'space-between',
+        paddingBottom: 24,
+    }),
+    {
+        flexWrap: 'wrap',
+        '@media': {
+            [mq.tabletOrBigger]: {
+                marginTop: 8,
+            },
+        },
+    },
+]);
+
+export const boxed = style([
+    sprinkles({display: 'flex'}),
+    {
+        isolation: 'isolate', // Needed to preserve border-radius with Video component and Safari
+    },
+]);
+
+export const interaction = style([
+    {
+        selectors: {
+            '&:hover': {
+                backgroundColor: vars.colors.backgroundContainerHover,
+                transition: '0.15s ease-in-out',
+            },
+            '&:active': {
+                backgroundColor: vars.colors.backgroundContainerPressed,
+                transition: '0.1s ease-in-out',
+            },
+        },
+    },
+]);
+
+export const cardContentStyle = style([
+    paddingX,
+    sprinkles({
+        display: 'flex',
+        paddingTop: 8,
+        paddingBottom: 24,
+    }),
+    {
+        '@media': {
+            [mq.desktopOrBigger]: {
+                paddingTop: 32,
+                paddingBottom: 32,
+            },
+        },
+    },
+]);
+
+export const minHeight = style([{minHeight: 216}]);
```

Comment:
don't use array if you only have one style
```suggestion
export const minHeight = style({minHeight: 216});
```